src/views/ProfilePage.jsx

```diff
@@ -1,8 +1,29 @@
 import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { login, logout } from '../store/userSlice'
+
 
 function ProfilePage() {
+
+  const user = useSelector((state) => state.user.user)
+  const dispatch = useDispatch()
+
   return (
-    <div>Profile page</div>
+    <div>Profile page
+      <button
+          onClick={() => dispatch(login())}
+        >
+          login
+        </button>
+      <div>{user}</div>
+      <button
+          onClick={() => dispatch(logout())}
+        >
```

Comment:
the button can be written on one line, it's a short component :) in general, just check that your lines are shorter than 100 characters